File: BackEnd_Tech/BackEnd_Tech/Models/HoaDonNhap.cs
using System;
using System.Collections.Generic;

namespace BackEnd_Tech.Models
{
    public partial class HoaDonNhap
    {
        public HoaDonNhap()
        {
            ChiTietHoaDonNhaps = new HashSet<ChiTietHoaDonNhap>();
        }

        public int Id { get; set; }
        public int? NhaCungCapId { get; set; }
        public int? UserId { get; set; }
        public decimal? TongTien { get; set; }
        public string? TrangThaiThanhToan { get; set; }
        public DateTime? CreatedAt { get; set; }

        public virtual NhaCungCap? NhaCungCap { get; set; }
        public virtual NhanVien? User { get; set; }
        public virtual ICollection<ChiTietHoaDonNhap> ChiTietHoaDonNhaps { get; set; }
    }
}

File: FontAdmin_Tech/src/app/Models/hoa-don-nhap.ts
export interface IHoaDonNhap {
    id?: number
    nhaCungCapId?: number
    userId?: number
    tongTien?: number
    trangThaiThanhToan?: string
    chiTietHoaDonNhaps?:
        | {
              sanPhamId?: number
              giaNhap?: number
              soLuong?: number
              thanhTien?: number
          }[]
        | null
}
